mods/ra/maps/soviet-13a/soviet13a.lua

```diff
@@ -0,0 +1,119 @@
+--[[
+   Copyright (c) The OpenRA Developers and Contributors
+   This file is part of OpenRA, which is free software. It is made
+   available to you under the terms of the GNU General Public License
+   as published by the Free Software Foundation, either version 3 of
+   the License, or (at your option) any later version. For more
+   information, see COPYING.
+]]
+Jeeps = { Jeep1, Jeep2 }
+JeepWaypoints = { JeepWaypoint1.Location, JeepWaypoint2.Location }
+OreAttackers = { OreAttack1, OreAttack2, OreAttack3, OreAttack4, OreAttack5, OreAttack6, OreAttack7, OreAttack8 }
+RadarSites = { Radar1, Radar2, Radar3, Radar4 }
+StartAttack = { StartAttack1, StartAttack2, StartAttack3, StartAttack4, StartAttack5 }
+ChronoDemolitionTrigger = { CPos.New(35,96), CPos.New(36,96), CPos.New(37,96), CPos.New(37,97), CPos.New(38,97), CPos.New(39,97) }
+OreAttackTrigger = { CPos.New(105,67), CPos.New(106,67), CPos.New(107,67), CPos.New(108,67), CPos.New(109,67), CPos.New(110,67) }
+
+Start = function()
+	Reinforcements.Reinforce(USSR, { "mcv" }, { MCVEntry.Location, DefaultCameraPosition.Location }, 5)
+
+	Utils.Do(Jeeps, function(jeep)
+		jeep.Patrol(JeepWaypoints, true, 125)
+	end)
+
+	Utils.Do(StartAttack, function(a)
+		IdleHunt(a)
+	end)
+
+	ChronoCam = Actor.Create("camera", true, { Owner = USSR, Location = Chronosphere.Location})
+end
+
+MissionTriggers = function()
+	Trigger.OnAllKilledOrCaptured(RadarSites, function()
+		USSR.MarkCompletedObjective(TakeDownRadar)
+		ChronoshiftAlliedUnits()
+	end)
+
+	Trigger.OnCapture(Chronosphere, function()
+		if not USSR.IsObjectiveCompleted(TakeDownRadar) then
+			Chronosphere.Kill()
```

Comment:
```suggestion
			Media.DisplayMessage(UserInterface.Translate("chrono-trap-triggered"), UserInterface.Translate("headquarters"))
			Chronosphere.Kill()
```

this should also trigger the message